src/app/screens/Home/index.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 
 import Navbar from '~components/Navbar';
 
+import BookList from './screens/BookList';
 import styles from './styles.module.scss';
 
 export default function Home() {
   return (
     <div className={`column ${styles.container}`}>
       <Navbar />
+      <div className={styles.content}>
```

Comment:
can the styles of this container div be inside the component?

Reply:
of course, but i had thought that it would be the layout for private routes in a future.
thats why I didnt include in the component